### PR TITLE
V2023.3.0  + dual protobuf

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by hmaarrfk** [comment]
thank you!

**Comment by hmaarrfk** [comment]
If this is making the builds too fragile for windows, we could try to leave the windows builds as they were and only collapse the UNIX build.

We did this for pytorch:
https://github.com/conda-forge/pytorch-cpu-feedstock/blob/main/recipe/conda_build_config.yaml#L19

**Comment by ilya-lavrenov** [comment]
> If this is making the builds too fragile for windows, we could try to leave the windows builds as they were and only collapse the UNIX build.
> 
> We did this for pytorch: https://github.com/conda-forge/pytorch-cpu-feedstock/blob/main/recipe/conda_build_config.yaml#L19

good idea

File: recipe/meta.yaml [hunk 1]
```diff
@@ -641,35 +643,19 @@ outputs:
...
-
-        # check that devices are found using OpenVINO C API
-        - $CC snippets/openvino_available_devices.c $(pkg-config --cflags --libs openvino) -Wl,-rpath,$PREFIX/lib -o openvino_devices_test  # [unix]
-        - ./openvino_devices_test  # [unix]
-
-        # check that frontends are found using OpenVINO C++ API
-        - cmake -G "Ninja Multi-Config" -S snippets -B snippets_openvino_{{ CONDA_PY }}_build
-        - cmake --build snippets_openvino_{{ CONDA_PY }}_build --config Release --verbose
-        - ./snippets_openvino_{{ CONDA_PY }}_build/Release/openvino_frontends_test  # [unix]
-        - .\\snippets_openvino_{{ CONDA_PY }}_build\\Release\\openvino_frontends_test.exe  # [win]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:672]: These seems seem to be duplicated from `libopenvino-dev` lets just see if removing them reduces the build time.

**Pull request merged**